package-res/resources/web/pentaho/data/filter/AbstractFilter.js

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "../../lang/Base",
+  "./_Element",
+  "../TableView",
+  "../../util/arg",
+  "require"
+  //"./Or",
+  //"./And",
+  //"./Not",
+], function(Base, Element, TableView, arg, require,  Or, And, Not) {
```

Comment:
I had to comment these three dependencies in order to get the tests to run, but I don't understand why.